tests/test_run_subproc.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+import builtins
+
+import pytest
+
+from xonsh.platform import ON_WINDOWS
+from xonsh.built_ins import run_subproc
+
+from tools import skip_if_on_windows
+
+
+@pytest.yield_fixture(autouse=True)
+def chdir_to_test_dir(xonsh_builtins):
```

Comment:
Better use an actual temp dir here instead. This is something we need to change many places in the test suite. 

```
def chdir_to_test_dir(tmpdir):
    with tmpdir.as_cwd():
        yield
```